eth/polar/polaris.go

```diff
@@ -148,3 +149,8 @@ func (pl *Polaris) StartServices() error {
 	}()
 	return nil
 }
+
+// SetGenesisBlock is a wrapper which sets the Genesis block in blockchain.
+func (pl *Polaris) SetGenesisBlock(genesisBlock *types.Block) error {
+	return pl.blockchain.SetGenesisBlock(genesisBlock)
```

Comment:
the blockchain should just take the genesis block as a constructor arg imo? if possible

Reply:
(maybe difficult cause cosmos)